api/src/main/java/org/apache/iceberg/actions/MigrateTable.java

```diff
@@ -41,6 +41,14 @@ public interface MigrateTable extends Action<MigrateTable, MigrateTable.Result>
    */
   MigrateTable tableProperty(String name, String value);
 
+  /**
+   * Sets the dropBackup property to drop backup table after a successful table Migration
+   *
+   * @param dropBackup flag to remove backup table
```

Comment:
Can be more descriptive, maybe "if true, removes the backup table"

Reply:
Done